pkg/smartcontract/rpcbinding/binding.go

```diff
@@ -519,6 +553,30 @@ func dropStdEvents(events []manifest.Event, std *standard.Standard) []manifest.E
 	return events
 }
 
+// dropNep24Types removes NamedTypes of NEP-24 from the config if they are used only once.
+func dropNep24Types(cfg binding.Config) binding.Config {
+	var targetTypeName string
+	if royaltyInfo, ok := cfg.Types["royaltyInfo"]; ok && royaltyInfo.Value != nil {
```

Comment:
Move the method name to a constant. Reuse this constant from `dropNep24Types` and from standard definition itself.